index.js

```diff
@@ -0,0 +1,214 @@
+    if (item.code === 'Tab') {
+      document.querySelector('textarea').value = `${document.querySelector('textarea').value.slice(0, cursor)}    ${document.querySelector('textarea').value.slice(cursor, document.querySelector('textarea').value.length)}`;
+      document.querySelector('textarea').selectionEnd = cursor + 4;
+    }
+    if (item.type === 'write') {
+      write(item);
+    }
+  } else if (item.code === 'Backspace' || item.code === 'Delete' || item.code === 'Tab' || item.code === 'Enter' || item.type === 'write') {
+    cursor = document.querySelector('textarea').selectionStart;
+    document.querySelector('textarea').value = document.querySelector('textarea').value.slice(0, document.querySelector('textarea').selectionStart) + document.querySelector('textarea').value.slice(document.querySelector('textarea').selectionEnd, document.querySelector('textarea').value.length);
+    document.querySelector('textarea').selectionStart = cursor;
+    document.querySelector('textarea').selectionEnd = cursor;
+    if (item.code !== 'Backspace' && item.code !== 'Delete') { enterSymbols(item); }
+  }
+}
+
+
+// PREPARATION
+
+function keyFill(language) {
+  localStorage.languageOfKeyboard = language;
+  document.querySelector('.keyboard').innerHTML = '';
+  objOfKeys.forEach((item) => {
+    const key = document.createElement('div');
+    key.className = `${item.size} ${item.code}`;
+    if (language === 'eng') {
+      if (item.view === 'double') {
+        key.innerHTML = `<p class="upLetter">${item.shiftEngValue}</p>`;
+        key.innerHTML += `<p class="centerLetter">${item.engValue}</p>`;
+      } else key.innerHTML = `<p class="centerLetter">${item.shiftEngValue}</p>`;
+    } else if (item.view === 'double') {
+      key.innerHTML = `<p class="upLetter">${item.shiftRuValue}</p>`;
+      key.innerHTML += `<p class="centerLetter">${item.ruValue}</p>`;
+    } else key.innerHTML = `<p class="centerLetter">${item.shiftRuValue}</p>`;
+    document.querySelector('.keyboard').append(key);
+  });
+}
+
+
+function createBase() {
+  const main = document.createElement('main');
+  document.querySelector('body').prepend(main);
+  const input = document.createElement('textarea');
+  document.querySelector('main').prepend(input);
+  const keyboard = document.createElement('section');
+  keyboard.className = 'keyboard';
+  document.querySelector('main').append(keyboard);
+}
+
+createBase();
+document.querySelector('textarea').focus();
+document.querySelector('textarea').setAttribute('placeholder', 'Keyboard designed for WINDOWS\nTo change the language click right shift and right alt!\nEnjoy ^__^');
+document.querySelector('body').setAttribute('onselectstart', 'return false');
+
+
+if (languageOfKeyboard === 'eng') {
+  keyFill('eng');
+} else {
+  keyFill('ru');
+}
+
+
+// START OF EVENTS
+
+document.querySelector('body').addEventListener('mousedown', (e) => {
```

Comment:
The function inside this addEventListener should be moved to a separate file (src/js/events.js) as I wrote above